test_regress/t/t_dpi_accessors.cpp

```diff
@@ -246,7 +246,7 @@ int main() {
     for (int i = 0; !Verilated::gotFinish() && (i < 4); i++) {
         dut->clk = 1 - dut->clk;
         a = 1 - (int)a_read();
-        a_write(a);
+        a_write((const svBitVecVal*)&a);
```

Comment:
Please use C++ style cast if possible.

Reply:
The rest of that file uses C style casts, so I just copied what was already there. I will have a skim through.

Reply:
I fixed all casts in that test in a separate commit in this PR.